Documentation/CodeSnippets/TypoScriptSyntax/Comments.rst.txt

```diff
@@ -0,0 +1,25 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: Extension examples, file Configuration/TypoScript/Syntax/Comments/setup.typoscript
+
+    # This is a comment
+    // This is a comment
+    /* This is a
+       multiline comment */
```

Comment:
I'm not sure if it was possible like that, at the end of a comment line.  
At least like this it's not possible to use "*/" inside a comment (at least without escaping)